cockatrice/src/chatview.cpp

```diff
@@ -168,8 +168,15 @@ void ChatView::appendMessage(QString message, QString sender, UserLevelFlags use
         } else
             from = 1;
     }
+
+    QTextCharFormat charFormat;
+    if (message.contains(QString::fromStdString(tabSupervisor->getUserInfo()->name()))) {
+        charFormat.setFontWeight(QFont::Bold);
+        charFormat.setForeground(Qt::red);
```

Comment:
Can this be done with a css class and the style decoupled so it's easier to change?